docs/fundamentals/vector-search.txt

```diff
@@ -0,0 +1,162 @@
+.. _laravel-vector-search:
+
+===================
+Atlas Vector Search
+===================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, semantic, text, embeddings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to perform searches on your documents
+by using the Atlas Vector Search feature. {+odm-long+} provides an API to
+perform Atlas Vector Search queries directly with your models. This guide
+describes how to create Atlas Vector Search indexes and provides
+examples of how to use the {+odm-short+} to perform searches.
+
+.. note:: Deployment Compatibility
+
+   You can use the Atlas Vector Search feature only when
+   you connect to MongoDB Atlas clusters. This feature is not
+   available for self-managed deployments.
+
+To learn more about Atlas Vector Search, see the :atlas:`Overview
+</atlas-vector-search/vector-search-overview/>` in the 
+Atlas documentation. The Atlas Vector Search API internally uses the
+``$vectorSearch`` aggregation operator to perform queries. To learn more about
+this operator, see the :atlas:`$vectorSearch
+</atlas-vector-search/vector-search-stage/#syntax>` reference in the Atlas
+documentation.
+
+.. note::
+   
+   You might not be able to use the methods described in
+   this guide for every type of Atlas Vector Search query.
+   For more complex use cases, create an aggregation pipeline by using
+   the :ref:`laravel-aggregation-builder`.
+   
+   To perform advanced full-text searches on your documents, you can use the
+   {+odm-long+} Atlas Search API. To learn about this feature, see
+   the :ref:`laravel-atlas-search` guide.
+
+.. _laravel-avs-index:
+
+Create an Atlas Vector Search Index
+-----------------------------------
+
+.. TODO in DOCSP-46230
+
+Perform Queries
+---------------
+
+In this section, you can learn how to use the Atlas Vector Search API in
+the {+odm-short+}. The {+odm-short+} provides the ``vectorSearch()``
+method as a query builder method and as an Eloquent model method. You
+can use the ``vectorSearch()`` method to run Atlas Vector Search queries
+on documents in your collections.
+
+You must pass the following parameters to the ``vectorSearch()`` method:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+	 - Type
+	 - Description
+
+   * - ``index``
+	 - ``string``
+	 - Name of the vector search index
+
+   * - ``path``
+	 - ``array`` or ``string``
```

Comment:
There is an error in this stage definition that I'm going to fix in https://github.com/mongodb/laravel-mongodb/pull/3263
```suggestion
	 - ``string``
```